### PR TITLE
fix link

### LINKED ISSUE #2522: [DOCS] Quickstart step 9 - link to icon search page
### Description

In Step 9 of the quickstart, it is mentioned that the user must grab icon definitions from react font awesome fa/si.  To improve UX, link to the icon search page on LinkFree.  Make sure to open in a new tab automatically so the user does not lose their place in the documentation

https://linkfree.eddiehub.io/icons

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by alokVerma749**
## Fixes Issue

Closes #2522

## Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Abstract Colorful Daily Affirmation Instagram Post

## Note to reviewers

now link opens to a new tab

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4250"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

File: pages/docs/quickstart.mdx [hunk 1]
```diff
@@ -79,7 +79,7 @@ To do this, you need a GitHub account. If you do not have one yet, you can creat
 | :---- | :----- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | name  | string | The display name for the button                                                                                                                                                      |
 | url   | string | Full url to the resource                                                                                                                                                             |
-| icon  | string | The icon you would like displayed. We support the following icon sets: **Fa** and **Si** from React Icons. You can search available [icons here](https://linkfree.eddiehub.io/icons) |
+| icon  | string | The icon you would like displayed. We support the following icon sets: **Fa** and **Si** from React Icons. You can search available <a href="https://linkfree.eddiehub.io/icons" target="_blank">icons here</a>  |
```
Comment (eddiejaoude) [inline pages/docs/quickstart.mdx:82]: Target blank is stripped by GitHub, suggest using markdown

```suggestion
| icon  | string | The icon you would like displayed. We support the following icon sets: **Fa** and **Si** from React Icons. You can search available [icons](https://linkfree.eddiehub.io/icons)  |
```

Comment (eddiejaoude) [reply pages/docs/quickstart.mdx:82]: Yes they suggest not to use `blank` also. We are staying within the same app, a new tab is not required

**Review by eddiejaoude — APPROVED** [review]
Thank you for your contribution 👍

**Pull request merged**